workflow/README.md

```diff
@@ -81,6 +80,8 @@ Steps of this type are used to configure sources which workflow manager will use
   ]
 }
 ```
+#### Limitations
+- We do not support chunk partitioning for sources which are not partitioned by date.
```

Comment:
that's a good point, let's create a ticket to add later

Reply:
We need to create a separate board for `rai-worlflow-manager`